vendor/CMakeLists.txt

```
@@ -2,5 +2,7 @@ if(NOT ENABLE_SYSTEM_GLM)
     add_subdirectory(glm)
 endif()
 add_subdirectory(hlslparser)
+if(NOT ENABLE_SYSTEM_PROJECTM_EVAL)
```

Just check for the target here.

```suggestion
if(NOT TARGET projectM::Eval)
```